pkg/controller/operandrequest/operandrequest_controller.go

```diff
@@ -389,12 +388,12 @@ func (r *ReconcileOperandRequest) listConfig(namespace string) (*operatorv1alpha
 				cscList.Items[0].Name+
 				". You need to leave one and delete the others")
 	}
+	reqLogger.Info("Found OperandConfig instance: " + cscList.Items[0].Name)
```

Comment:
I think we should use klog to manage all of the log info here, opened an issue at https://github.com/IBM/operand-deployment-lifecycle-manager/issues/138